templates/docs/examples/patterns/notifications/_variants.html

```diff
@@ -0,0 +1,215 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Notifications / Variants{% endblock %}
+
+{% block standalone_css %}patterns_notifications{% endblock %}
+
+{% block style %}
+<link rel="stylesheet" href="{{ versioned_static('build/css/standalone/patterns_icons-additional.css') }}" />
+{% endblock %}
+
+{% block content %}
+
+<h5>Borderless</h5>
```

Comment:
Because it's just a HTML partial, you don't need all the jinja template tags `{% %}` that examples have.

It can simply start with `<h5>Borderless</h5>` and end with `</script>`, without any blocks etc.